### PR TITLE
Update 01-07

### LINKED ISSUE #8: 🐛 [Bug] Missing viewport
## 🐛 Bugreport

Google Pagespeed Insights report missing viewport: https://pagespeed.web.dev/report?url=http%3A%2F%2Fit4c.dev%2F

### LINKED ISSUE #10: 🔧 [Refactor] Link tech in readme

### LINKED ISSUE #19: 🚀 [Feature] Startpage content

### PR DESCRIPTION
**Pull request by ulfgebhardt**
## 🍰 Pullrequest

Update from the 07.01.2023

image

### Issues

- fixes #10
- fixes #19
- fixes #8

### Stuff without ticket
- removed todos as they are either done or available as issue

### Todo

- [X] None

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,66 +1,37 @@
 # IT4C
 
-IT Team for Change website
+**IT Team for Change**, in short **IT4C**, is a software developer team building software together. Our focus are out-of-the-box projects and disruptive civil tech to contribute to the inevitable change of our society.
```
Comment (Tirokk) [inline docs/index.md:3]: The wording `disruptive` can be discussed, like I said before.
But lets take it for now.

File: docs/index.md [hunk 2]
```diff
@@ -1,66 +1,37 @@
...
 
-# IT4C Website
+We believe in a working process that has the right amount of authority to make sensible decisions and coordinate us to reach our goals. The leadership may shift between team members based on expertise and availability. Furthermore we believe that everyone who loves doing his job is more effective then when being forced to. We make sure every member has a the freedom to choose how to work.
```
Comment (Tirokk) [inline docs/index.md:5]:
```suggestion
We believe in a working process that has the right amount of authority to make sensible decisions and coordinate us to reach our goals. The leadership may shift between team members based on expertise and availability. Furthermore we believe that everyone who loves doing his job is more effective then when being forced to. We make sure every member has a great freedom to choose how to work.
```

File: docs/index.md [hunk 3]
```diff
@@ -1,66 +1,37 @@
...
 
-- [x] Find a web address
-<!-- textlint-disable no-todo -->
-- [ ] Find a Logo
-<!-- textlint-enable no-todo -->
+Naturally we are a remote team spread throughout Europe (sometimes even further). The core of the team works together as of 2019 - see [history](./history.md). Our common denominator besides our love for social projects and the computer technology is the German language, even tho most of the professional work is done in English.
```
Comment (Tirokk) [inline docs/index.md:7]: > Our common denominator besides our love for social projects and the computer technology is the German language, even tho most of the professional work is done in English.

Why is this neccessary?

File: docs/index.md [hunk 4]
```diff
@@ -1,66 +1,37 @@
...
-- Gradido
-
-Projekte werden bebildert, der Techstack aufgeführt, sowie beteiligte Personen aus dem Team.
-Idealerweise würden diese Daten von github kommen (später)
-
-## Organisationen
+<!-- ## Organisationen -->
 <!-- textlint-disable period-in-list-item -->
-- busFaktor() e.V.
+<!-- - busFaktor() e.V. -->
```
Comment (Tirokk) [inline docs/index.md:36]: Why you have commented this?

**Review by Tirokk — CHANGES_REQUESTED** [review]
Hey cool! 🚀💫

Ich hab noch Vorschläge und Fragen …

File: docs/index.md (hunk 2 above)
Comment (ulfgebhardt) [reply docs/index.md:5]: this is wrong english.

I had `every member has a maximum of freedom` but the linter didnt like that

File: docs/index.md (hunk 3 above)
Comment (ulfgebhardt) [reply docs/index.md:7]: I don't understand the question

File: docs/index.md (hunk 4 above)
Comment (ulfgebhardt) [reply docs/index.md:36]: this is just an idea - it is unclear if we should support organisations on this page and which are orgas we want to feature

File: docs/index.md (hunk 2 above)
Comment (ulfgebhardt) [reply docs/index.md]: Please check this again, I made a new phrasing

File: docs/index.md [hunk 5]
```diff
@@ -1,66 +1,37 @@
 # IT4C
 
-IT Team for Change website
+**IT Team for Change**, in short **IT4C**, is a software developer team building software together. Our focus are out-of-the-box projects and disruptive civil tech to contribute to the inevitable change of our society.
 
-# IT4C Website
+We believe in a working process that has the right amount of authority to make sensible decisions and coordinate us to reach our goals. The leadership may shift between team members based on expertise and availability. Furthermore we believe that everyone who loves doing his job is more effective then when being forced to. We make sure every member has the upmost possible freedom how to work.
```
Comment (Tirokk) [inline docs/index.md:5]: 👍🏼

**Review by Tirokk — APPROVED** [review]
👍🏼

File: docs/index.md (hunk 2 above)
Comment (Tirokk) [reply docs/index.md]: Right

File: docs/index.md (hunk 3 above)
Comment (Tirokk) [reply docs/index.md:7]: Translated this means:

> Unser gemeinsamer Nenner ist neben der Liebe zu sozialen Projekten und der Computertechnik die deutsche Sprache, auch wenn der Großteil der beruflichen Arbeit auf Englisch erfolgt.

If our common denominator is the German language that sounds exclusive to people of other languages. Or what you think?

**Review by Tirokk — APPROVED** [review]
💫

**Pull request merged**